### DISCUSSION TITLE
Why are citations hidden behind the closed beta version of the API?

### DISCUSSION THREAD

**Discussion by Weilin37** [perplexityai/api-discussion#40] [General]
If the whole point of this tool is to provide accurate answers searched online, why are the citations not available, and that I have to request for it to be enabled via the closed beta form?

**Comment by uluhonolulu** [comment]
Where do you see the list of closed beta features?

**Reply by rafs2006** [reply]
Hey @uluhonolulu! You can check the parameters here: https://docs.perplexity.ai/api-reference/chat-completions
![image](https://github.com/user-attachments/assets/82fb7043-95b5-4a83-8089-9b4180194093)

**Comment by liam-twist** [comment]
Any update here?
We're considering using the perplexity AI api but citations are crucial to our use case. We've just applied for the closed beta but would prefer if citations were made generally available instead.
Does anyone have an update on when citations will be available outside the closed beta?

**Comment by aarashy** [comment]
Hey, we have on good authority that citations access will be public soon.

**Comment by aarashy** [comment]
And its out!
